packages/app-server-issuer/src/exchange-provider/ExchangeProviderManager.ts

```diff
@@ -72,27 +71,31 @@ export class ExchangeProviderManager {
         }),
       },
       this.context.ckitProvider,
-      this.providerAddress,
+      this.context.txSigner!.getAddress(),
     );
-    const unsignedTx = await builder.build();
-    const tx = await this.context.txSigner.seal(unsignedTx);
-    const hash = await this.context.ckitProvider.sendTransaction(tx);
-    await this.context.ckitProvider.waitForTransactionCommitted(hash);
+    try {
+      const unsignedTx = await builder.build();
+      const tx = await this.context.txSigner.seal(unsignedTx);
```

Comment:
```suggestion
      const tx = await this.context.txSigner.partialSeal(unsignedTx);
```

Reply:
> 

For manager, cells creation could only be signed by acp signer, no other one needs to provider extra cells for input, I think. Maybe you mean in handler I should change the code like this, is it right?